Header header
string before
string after